vm/src/hint_processor/builtin_hint_processor/hint_utils.rs

```diff
@@ -124,14 +124,14 @@ pub fn get_reference_from_var_name<'a>(
 }
 
 pub fn get_constant_from_var_name<'a>(
-    var_name: &'static str,
+    var_name: &'a str,
```

Comment:
```suggestion
    var_name: &str,
```
I think that should actually be enough.